design-proposals/cluster-orch-integrating-k8s-into-emt.md

```diff
@@ -0,0 +1,253 @@
+# Design Proposal: Integrating Kubernetes into Edge Microvisor Toolkit
+
+Author(s): Hyunsun Moon
+
+Last updated: 5/2/25
+
+## Abstract
+
+The Edge Microvisor Toolkit (EMT) is an operating system specifically designed for hosting edge workloads, streamlining traditional general-purpose operating systems by including only the essential components needed to run container-based applications. Our experience from previous releases has demonstrated that EMT's design principles—image-based deployment and immutable root filesystem—enhance the reliability and consistency of cluster creation compared to even well-maintained general-purpose operating systems, such as Ubuntu.
```

Comment:
btw on the long term will be these the only image we plan to support for EMT/EMT-S? cc @krishnajs